deploy/staging/shared-environment.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: shared-environment
+data:
+  RAILS_ENV: "production"
+  RAILS_WEB_CONCURRENCY: "5"
```

Comment:
What does this `RAILS_WEB_CONCURRENCY` environment variable do?

I see in the staging Kubernetes deployment config, we've given it 2 replicas (i.e. 2 load balanced pods serving traffic)

Should these values match?

Reply:
Looks like we don't need this environment variable, nor the associated `config.connection_pool_size` key in `application.rb`